code/obj/item/organs/organ_parent.dm

```diff
@@ -341,3 +351,12 @@
 			return 1
 		else
 			return 0
+
+	proc/organ_fix_colors(var/hex)
```

Comment:
document what this is supposed to do please

Reply:
I'm not completely sure what it does, it was part of the lizard mutantrace datum and copied over so it'd work on organs. From what I can tell, it takes a hex color, converts it to RGB, clamps the values, then converts it back to hex.

I'll stick a comment on both of the procs explaining that.